src/RazorConsole.Core/AppHost.cs

```diff
@@ -55,6 +55,7 @@ public static IHostApplicationBuilder UseRazorConsole<TComponent>(
         Action<IHostApplicationBuilder>? configure = null)
         where TComponent : IComponent
     {
+        RuntimeEncoding.EnsureUtf8();
         RegisterDefaults<TComponent>(hostBuilder.Services);
```

Comment:
Let's check if utf-8 is already enabled by default if applicable in spectre.console?